src/LinuxTracing/PerfEventRecords.h

```diff
@@ -40,7 +42,6 @@ struct __attribute__((__packed__)) perf_event_fork_exit {
 
 // This struct must be in sync with the SAMPLE_REGS_USER_ALL in PerfEventOpen.h.
```

Comment:
That's weird, it wasn't valid before. It is actually in sync now, before the struct had `abi` which wasn't present in the flag.

Reply:
Ah. `SAMPLE_REGS_USER_ALL` is a flags field. I guess you could argue that it was also in sync before because `abi` is not optional when capturing registers is enabled.

I guess it's fine to leave it as-is.